plugins/woocommerce-blocks/assets/js/blocks/product-filters/components/initial-disabled/index.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+/**
+ * The reason for this component instead of using core/disabled is, the Disabled
+ * component messes up the focus on inner blocks. For example, when a there is
+ * an heading block nested inside, the text cursor, which indicates the editable
+ * area, isn't visible when focus on the heading block.
+ *
+ * This component only use CSS to control the selecte behavior of inner blocks,
+ * which fix the issues like mentioned above.
+ *
+ * Limitation: being a static component comes with a limitation, this component
+ * is meant to put directly inside the block wrapper element that holds block
+ * props.
+ */
+const InitialDisabled = ( {
+	children,
+}: {
+	isDisabled: boolean;
+	children: React.ReactNode;
+} ): JSX.Element => (
+	<div className="wc-component-initial-disabled">
+		<div className="wc-component-initial-disabled-overlay" />
+		{ children }
+	</div>
+);
+
+export default InitialDisabled;
```

Comment:
It seems to be common in our codebase to use named exports when exporting elements from a file, likely because it helps IntelliSense automatically import those elements into other files, and it also makes them easier to search for within the codebase.

```suggestion
export const InitialDisabled = ( {
	children,
}: {
	isDisabled: boolean;
	children: React.ReactNode;
} ): JSX.Element => (
	<div className="wc-component-initial-disabled">
		<div className="wc-component-initial-disabled-overlay" />
		{ children }
	</div>
);
```

Reply:
I removed the default export per your suggestion.